framework/packages/abstract-core/src/objects/voting.rs

```diff
@@ -126,7 +127,7 @@ impl<'a> SimpleVoting<'a> {
     pub fn new_proposal(
         &self,
         store: &mut dyn Storage,
-        end: Expiration,
+        end: Timestamp,
```

Comment:
What's the reasoning for doing only timestamps?

Reply:
Since we now have implicit transition to veto period, it will be simply easier to maintain one type of durations.
For example: we have veto period as 5 blocks and vote duration 50 seconds, when do we transition then? Giving a user to chose one option and stick with it would be also annoying, because some challenge contract instances would be only block-based, other timestamp based